lib/ASRouter.jsm

```diff
@@ -43,6 +44,8 @@ const MAX_MESSAGE_LIFETIME_CAP = 100;
 const LOCAL_MESSAGE_PROVIDERS = {OnboardingMessageProvider, CFRMessageProvider, SnippetsTestMessageProvider};
 const STARTPAGE_VERSION = "6";
 
+const ADDONS_API_URL = "https://services.addons.mozilla.org/api/v3/addons/addon";
```

Comment:
Retrieving the data should have used AddonRepository.jsm so there would be no need to hard code a url here.  At the very least, there should be a followup to remove this and use the pref.  Same with CFRPageActions.jsm.

Reply:
@mixedpuppy the code was refactored and we're no longer making this call.
https://github.com/mozilla/activity-stream/blob/master/lib/OnboardingMessageProvider.jsm